src/Lumper.UI/ViewModels/Tasks/ChangeTextureTaskViewModel.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using Lumper.Lib.Tasks;
+
+namespace Lumper.UI.ViewModels.Tasks;
+
+/// <summary>
+///     ViewModel for ChangeTextrue Task
+/// </summary>
+public partial class ChangeTextureTaskViewModel : TaskViewModel
+{
+    public ChangeTextureTaskViewModel(ChangeTextureTask task)
+        : base(task)
+    {
+        try
+        {
+            foreach (var item in task.Replace)
+                ReplaceItems.Add(new ChangeTextureReplaceItem(this, task, item));
+            foreach (var item in task.ReplaceRegex)
+                RegexItems.Add(new ChangeTextureRegexItem(this, task, item));
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine(ex.Message);
+        }
+    }
+
+    public ObservableCollection<ChangeTextureReplaceItem> ReplaceItems { get; private set; } = new();
+    public ObservableCollection<ChangeTextureRegexItem> RegexItems { get; private set; } = new();
+
+    public void AddReplaceItem()
+    {
+        if (Task is ChangeTextureTask c)
+        {
+            var replaceItem = new ChangeTextureReplaceItem(this, c);
+            if (replaceItem.Add())
+                ReplaceItems.Add(replaceItem);
+        }
+    }
+
+    public void AddRegexItem()
+    {
+        if (Task is ChangeTextureTask c)
+        {
+            var tmp = new ChangeTextureRegexItem(this, c);
+            if (tmp.Add())
+                RegexItems.Add(tmp);
```

Comment:
Oh sorry, shouldn've brought up the above thing with renaming `tmp` here as well